ci/spark-script/spark-connect-server.sh

```diff
@@ -0,0 +1,21 @@
+set -ex
+
+ICEBERG_VERSION=1.3.1
+SPARK_VERSION=3.4.1
+
+PACKAGES="org.apache.iceberg:iceberg-spark-runtime-3.3_2.12:$ICEBERG_VERSION,org.apache.hadoop:hadoop-aws:3.3.2"
```

Comment:
The spark version in the package version is not consistent with the spark image?